uint32 info_time_encoder
bool info_emergency
int16 cmd_left_speed
int16 cmd_right_speed
int16 info_left_encoder
int16 info_right_encoder
uint8 info_brush_Level
uint8 cmd_brush_Level
uint8 info_brush_Lift
uint8 cmd_brush_Lift
uint8 info_water_Spray
uint8 cmd_water_Spray
uint8 info_wiper_Lift
uint8 cmd_wiper_Lift
uint8 info_vacuum
uint8 cmd_vacuum
bool cmd_valve
bool info_valve
uint8 info_clear_waterl
uint8 info_sewage_waterl